www/src/components/LayoutGenerator.tsx

```diff
@@ -13,59 +13,61 @@ export interface IColumn {
   onChangeOffset: Function,
 }
 
-const Column = ({
+function Column({
```

Comment:
This refactoring is related to: `ESLint: Function component is not a function declaration(react/function-component-definition)`